operator_tracker/src/main.rs

```diff
@@ -0,0 +1,178 @@
+use std::sync::Arc;
+
+use axum::body::Body;
+use axum::extract::Path;
+use axum::extract::State;
+use axum::http::StatusCode;
+use axum::response::IntoResponse;
+use axum::routing::get;
+use axum::routing::post;
+use axum::Json;
+use axum::Router;
+use ethers::providers::{Http, Provider, ProviderExt};
+use ethers::types::Address;
+use log::debug;
+use log::info;
+use operator_tracker::create_or_update_operator_version;
+use operator_tracker::serialize_or_err;
+use operator_tracker::{OperatorVersionPayload, RegistryCoordinator, RegistryCoordinatorContract};
+use sqlx::postgres::{PgPool, PgPoolOptions};
+use sqlx::Executor;
+
+// The RegistryCoordinator contract
+// This is the contract that the service will interact with to get the status of the operator
+// The response is a uint8 that represents the status of the operator
+// 0 - Never registered
+// 1 - Registered
+// 2 - Deregistered
+
+#[derive(Clone)]
+struct AppState {
+    pool: PgPool,
+    registry_coordinator: RegistryCoordinatorContract,
+}
+
+/// The main entry point for the application
+/// This function initializes the database, the Ethereum provider, and the RegistryCoordinator contract
+#[derive(argh::FromArgs)]
+struct Args {
+    /// path to the .env file
+    #[argh(option)]
+    env_file: Option<String>,
+}
+
+#[tokio::main]
+async fn main() {
+    let args: Args = argh::from_env();
+
+    if let Some(env_file) = args.env_file {
+        dotenv::from_filename(env_file).expect("Failed to load env file");
+    } else {
+        dotenv::dotenv().expect("Failed to load env file");
+    }
+
+    env_logger::init();
+
+    let database_url = &std::env::var("DATABASE_URL").expect("DATABASE_URL must be set");
+
+    let rpc_url = &std::env::var("RPC_URL").expect("RPC_URL must be set");
+
+    let registry_coordinator_addr = &std::env::var("REGISTRY_COORDINATOR_ADDRESS")
+        .expect("REGISTRY_COORDINATOR_ADDRESS must be set");
```

Comment:
We should consider reading this value from `alignedlayer_deployment_output.json`

Reply:
I don't think its important, we would still need to pass path to the deployment output file, and this way we avoid the extra complexity of parsing the file